bpf/lib/nodeport.h

```diff
@@ -62,12 +62,15 @@ static __always_inline bool nodeport_uses_dsr(__u8 nexthdr __maybe_unused)
 static __always_inline bool
 bpf_skip_recirculation(const struct __ctx_buff *ctx __maybe_unused)
 {
-	/* From XDP layer, we do not go through an egress hook from
-	 * here, hence nothing to be skipped.
+	/* We can't use ctx_is_skb() here because tc_index field
+	 * is not defined in XDP layer.
```

Comment:
nit: Lets leave the original comment at it's prior location. You are saying that you get a compilation error below when you just do ctx_is_skb()?

Reply:
rest lgtm